jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1392,6 +1392,14 @@
       (pushback (send *baxter* arm :inverse-kinematics coords-tmp :rotation-axis t) avs)
       (send self :angle-vector-sequence avs :fast nil 0 :scale 4.0)
       ))
+  (:wait-interpolation-until-grasp
+    (arm)
+    (while (send self :interpolatingp)
+           (if (send self :graspingp arm)
+             (progn
+               (ros::ros-info "[:wait-interpolation-until-grasp] Grasping detected. Cancel angle vector: ~a" arm)
+               (send self :cancel-angle-vector))))
+    (send *baxter* :angle-vector (send self :state :potentio-vector)))
```

Comment:
hmm...
it is not what i want to know, but i got the reason.

critical example is as below.
we can keep angle-vector by `variable` and apply it while `*baxter*`'s angle-vecter is totally different.
then, when we use `wait-interpolation-until-grasp`, which synchronized `*baxter*` and `*ri*`, `*baxter*`'s posture will be overwritten.

```
$ (setq av (send *baxter* :angle-vector))
$ (send *baxter* :fold-pose-back)
$ (send *ri* :angle-vector av)
$ (send *ri* :wait-interpolation-until-grasp)
```